test/integration/default/controls/module_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+OverrideDefaultModules = input('pillar').dig('proftpd', 'OverrideDefaultModules')
+
+if OverrideDefaultModules == True 
+  control 'Module configuration' do
+    title 'should match desired lines'
+
+    describe file('/etc/proftpd/modules.conf') do
+      its('content') { should include '## File managed by Salt' }
```

Comment:
Can you add some more tests, like validating a default module is present, and another is absent?

Reply:
I add 3 tests:

1. default value true
2. default value overload false
3. new value true

Reply:
LGTM :+1: